users/mocks/users.go

```diff
@@ -89,7 +89,14 @@ func (urm *userRepositoryMock) RetrieveByID(ctx context.Context, id string) (use
 	return val, nil
 }
 
-func (urm *userRepositoryMock) Members(ctx context.Context, groupID string, offset, limit uint64, gm users.Metadata) (users.UserPage, error) {
+func (urm *userRepositoryMock) RetrieveAll(ctx context.Context, offset, limit uint64, email string, um users.Metadata) (users.UserPage, error) {
```

Comment:
Why don't we mock this method? Do we miss some tests?

Reply:
Yes, some service tests are missing. I added a mock.

Reply:
Can we add an issue for the tests?

Reply:
Here: https://github.com/mainflux/mainflux/pull/1262/files#diff-28ebac99dae6f7fda6ef1772531588fc3e201a5bac3a41c3ea0c760989edb96aR152